lang/src/org/partiql/lang/ast/AggregateCallSiteListMeta.kt

```diff
@@ -15,11 +15,12 @@
 package org.partiql.lang.ast
 
 import com.amazon.ion.*
+import org.partiql.lang.domains.PartiqlAst
 
 /**
  * Contains references to each of the aggregate call-sites in a given [Select].
  */
-data class AggregateCallSiteListMeta(val aggregateCallSites: List<CallAgg>): Meta {
+data class AggregateCallSiteListMeta(val aggregateCallSites: List<PartiqlAst.Expr.CallAgg>): Meta {
```

Comment:
This is the crux of the main issue. Need to determine if this `Meta` should store a `CallAgg` in the PIG/PartiqlAst form or ExprNode form. Depending on what we decide, we would have to do the conversion in the evaluator (before [L45](https://github.com/partiql/partiql-lang-kotlin/blob/d5b2152cb04ba65636c66294d4dd447187cf3f81/lang/src/org/partiql/lang/eval/visitors/AggregateSupportVisitorTransform.kt#L45)) or in the VisitorTransform (current version).

Reply:
The `PartiqlAst` form here is desirable.